user/deployment/npm.md

```diff
@@ -16,6 +16,10 @@ publishes to npmjs.com, however if you have a `publishConfig.registry` key in yo
 A minimal `.travis.yml` configuration for publishing to [npmjs.com][npmjs] with npm version 2+ looks like:
 
 ```yaml
+language: node_js
+node_js:
+  - "11.13"
```

Comment:
```suggestion
  - "11.16"
```
thank you so much @martin-displayr for adding this. may I suggest to use the current LTS version instead?